src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -1,12 +1,24 @@
 package core.basesyntax;
 
+import java.time.DateTimeException;
+import java.time.Instant;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
 import java.util.Optional;
 
 public class JavaDateTimeApi {
+    private static final DateTimeFormatter CUSTOM_DATE = DateTimeFormatter
+            .ofPattern("dd MMM yyyy", Locale.ENGLISH);
+    private static final DateTimeFormatter MONTH_WRITTEN = DateTimeFormatter
```

Comment:
```suggestion
    private static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter
```
you may change this to another name, let's just make it clearer

Reply:
done